include/SFML/Window/WindowBase.inl

```diff
@@ -57,19 +44,11 @@ struct DelayOverloadResolution
 
 
 ////////////////////////////////////////////////////////////
-template <typename... Ts>
-void WindowBase::handleEvents(Ts&&... handlers) // NOLINT(cppcoreguidelines-missing-std-forward)
+template <typename... Handlers>
+void WindowBase::handleEvents(Handlers&&... handlers) // NOLINT(cppcoreguidelines-missing-std-forward)
 {
-    static_assert(sizeof...(Ts) > 0, "Must provide at least one handler");
-
-    // Disable misc-const-correctness for this line since clang-tidy
-    // complains about it even though the code would become uncompilable
-
-    // NOLINTNEXTLINE(misc-const-correctness)
-    priv::OverloadSet overloadSet{std::forward<Ts>(handlers)..., [](const priv::DelayOverloadResolution&) { /* ignore */ }};
-
     while (const std::optional event = pollEvent())
-        event->visit(overloadSet);
+        event->visit(handlers..., [](const priv::DelayOverloadResolution&) { /* ignore */ });
```

Comment:
`handlers...` should be forwarded here, and `NOLINT(cppcoreguidelines-missing-std-forward)` should be removed